cmd/ingress-merge/main.go

```diff
@@ -72,6 +88,30 @@ func main() {
 		"Process ingress resources with this `kubernetes.io/ingress.class` annotation.",
 	)
 
+	rootCmd.Flags().String(
+		"ingress-selector",
+		"",
+		"Process ingress resources with labels matching this selector string.",
+	)
+
+	rootCmd.Flags().String(
+		"configmap-selector",
+		"",
+		"Process configmap resources with labels matching this selector string.",
+	)
+
+	rootCmd.Flags().StringArray(
+		"ingress-blacklist",
```

Comment:
I think the argument would be better named `ingress-watch-ignore`. Same goes for configmaps & related fields.

Reply:
Done. Now it's --ingress-watch-ignore/IngressWatchIgnore/ingressWatchIgnore same with ConfigMaps.